<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>webshellGroup</class>
 <widget class="QWidget" name="webshellGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>5</number>
   </property>
   <item row="0" column="2">
    <widget class="QPushButton" name="btn_deletegroup">
     <property name="toolTip">
      <string>删除分组</string>
     </property>
     <property name="text">
      <string>删除分组</string>
     </property>
     <property name="icon">
      <iconset resource="../../qrc/img.qrc">
       <normaloff>:/webshell/groupDel.png</normaloff>:/webshell/groupDel.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="btn_addgroup">
     <property name="toolTip">
      <string>添加分组</string>
     </property>
     <property name="text">
      <string>新建分组</string>
     </property>
     <property name="icon">
      <iconset resource="../../qrc/img.qrc">
       <normaloff>:/webshell/groupAdd.png</normaloff>:/webshell/groupAdd.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="btn_renamegroup">
     <property name="toolTip">
      <string>重命名分组</string>
     </property>
     <property name="text">
      <string>重命名分组</string>
     </property>
     <property name="icon">
      <iconset resource="../../qrc/img.qrc">
       <normaloff>:/webshell/groupRename.png</normaloff>:/webshell/groupRename.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTreeWidget" name="tree_main">
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../qrc/img.qrc"/>
 </resources>
 <connections/>
</ui>
